stories/atoms/LoadingDots.stories.js

```diff
@@ -0,0 +1,40 @@
+import { Title, Description, Primary, Source, ArgsTable } from '@storybook/addon-docs';
+import LoadingDots from '../../components/atoms/LoadingDots';
+
+const sizeOptions = ['', 'sm', 'lg'];
```

Comment:
Is it expected that the first option is empty ?

Reply:
yes, it makes it so you can test applying no option (in this case, if you don't pass a size props, it'll render as what i'd call medium)